.github/workflows/phpunit.yml

```diff
@@ -25,16 +25,21 @@ jobs:
           --health-timeout 5s
           --health-retries 3
 
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       fail-fast: false
       matrix:
         php: ['7.2', '7.3', '7.4']
```

Comment:
Looks like we can now remove the `experimental` flag and add 8.0 to the list by default.

Reply:
I would like to keep the `experimental` flag for the moment.